### DISCUSSION TITLE
How to crop at exact dimensions please?

### DISCUSSION THREAD

**Discussion by le-khang** [photopea/photopea#5858] [General]
Hi Photopea team, thank you for this amazing website.

Recently, I have switched to Photopea instead of Photoshop for quick edits, and it's been great. I usually crop my images to 1024x1024 or 512x512, but the crop tool doesn't have the option to enter the exact pixel dimensions (like we can in Photoshop). As a result, I have to choose "Fixed ratio" >>  select "1:1" >> manually adjust the tool (1028 > 1016 > 1021 > 1024x1024)

Is there a quicker way to do this? Alternatively, could you please enable the option for users to enter the specific dimensions they need? Thank you very much.

**Comment by camoody1** [comment]
@le-khang Choose "Fixed Size" instead of "Fixed Ratio". Then, type in your desired cut size in the two fields next to the droplist.

**Reply by le-khang** [reply]
Thank you very much Camoody1. It works.

**Reply by camoody1** [reply]
Wonderful! Glad I could be of assistance. 🙂

**Comment by photopea** [comment]
Could you tell us how do you do it in Photoshop?

I am pretty certain that it should be possible to do in Photopea in the exactly the same way. So I am confused.

**Reply by le-khang** [reply]
Thank you for your quick reply. I followed Camoody1's instruction and it works. I didn't try all of your options.

**Comment by photopea** [comment]
If there is any other way to do it in Photoshop, I want to make it work the same in Photopea, so just tell us what you did in Photoshop.

**Reply by PebbleCat-Studios** [reply]
Don't you own a copy of photoshop? I would think you would have one to reference for the creation of Photopea.

**Reply by photopea** [reply]
Yes, I do have access to Adobe Photosohp. But the person says that there is a feature of Photoshop which does not exist in Photopea (i.e. a feature of Photoshop that I have never seen or heard of), so I want some proof that such a feature exists (e.g. where to find it in Photoshop).

**Reply by PebbleCat-Studios** [reply]
Oh, ok. :3
